website/components/remote-plugin-docs/server.js

```diff
@@ -54,7 +54,7 @@ async function generateStaticProps({
   // (current options are "Official" or "Community")
   function mdxContentHook(mdxContent) {
     if (pluginTier) {
-      const tierMdx = `<PluginTierLabel tier="${pluginTier}" />\n\n`
+      const tierMdx = `<br/><PluginTierLabel tier="${pluginTier}" />\n\n`
```

Comment:
This adds the spacing that we wanted to achieve with the `:global` styles, but without the layout shift on load. 

I'm open to other approaches here, part of me wishes this spacing was in CSS rather than markup, but this felt like a pragmatic solution given that `<PluginTierLabel />` itself is used in other contexts, so should not have built-in surrounding margin.

Reply:
Only other thing I can think of is passing a style prop to the component, but honestly this is fine